spec/system/calendar_spec.rb

```diff
@@ -25,7 +25,7 @@
       expect { click_on('Save') }.to change { calendar.calendar_events.count }.by(1)
 
       # Wait until ajax call is back
-      assert_selector('td[style]')
+      assert_selector('td[style*="background-color"]')
```

Comment:
Saw this failed on master, always worth trying to make the selector more fussy to see if that fixes it!